spec/lib/submission_builder/ty2022/states/return_header_spec.rb

```diff
@@ -22,7 +22,8 @@
           intake.direct_file_data.mailing_zip = mailing_zip
           intake.direct_file_data.tax_return_year = tax_return_year
           allow(EnvironmentCredentials).to receive(:irs).with(:efin).and_return efin
-          allow(EnvironmentCredentials).to receive(:irs).with(:sin).and_return sin
+          software_id = StateFile::StateInformationService.software_id_key(state_code).to_sym
+          allow(EnvironmentCredentials).to receive(:irs).with(software_id).and_return sin
```

Comment:
can we check in the test that the `SoftwareID` gets filled in with the correct value here?